src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -160,35 +165,46 @@ public void testTaskToClientMapping() throws Exception {
     sinkTask1.stop();
   }
 
+  @Mock Logger logger = Mockito.mock(Logger.class);
+
+  @InjectMocks @Spy
+  private LoggerHandler loggerHandler = Mockito.spy(new LoggerHandler(this.getClass().getName()));
+
+  @InjectMocks private SnowflakeSinkTask sinkTask1 = new SnowflakeSinkTask();
+
```

Comment:
can you move this to the top?

Reply:
These arent meant to be used in any other part of the test, so I moved them here for readability. They do have to be declared outside the method for mockito reasons.

Now that they're out here, i do see how its weird to have these variables declared mid test. I'll move them to the top and add a comment about it

Reply:
Also, just realized we are mocking the behavior in IT, can we instead move this to Unit Tests? [Seems like an anti pattern to me]

And IMO, we dont need to mock any of the behaviors here, I think your test would work if you remove Spy and inject mocks. 

Other way would be to just move out the logs test to its own class - Sorry didnt realize this earlier in your logs changes.

Reply:
yeah it prob should be its own class. Wasn't sure where to stick it back then because I didnt see a SnowflakeSinkTaskForStreamingTest. Can't remove the Spy or InjectMocks, I spent way too long figuring it out because the logger is a static final object so this is a not great work around for that. At the same time its testing too many methods for me to comfortably call it a unit test, so maybe we need another 'category' of testing conceptually. regardless for this PR i'll move it to a SnowflakeSinkTaskTest file in the streaming folder

Reply:
At the same time i dont think this test is strictly necessary, but i would like some form of logging testing i guess? Not that important but makes me feel better about any logging changes

Reply:
Ack, moving to its own class sounds good to me! Thanks